packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -449,6 +449,21 @@ export const schema: SchemaType<DbTag> = {
   'canVoteOnRels.$': {
     type: String,
   },
+
+  subforumShortformPostId: {
```

Comment:
It would be good to have a form component to edit this, but I don't think we should do it this way anyway so I've left it for now